exp/api/v1alpha3/azuremanagedcontrolplane_types.go

```diff
@@ -55,7 +57,7 @@ type AzureManagedControlPlaneSpec struct {
 	// +kubebuilder:validation:Enum=Calico;Azure
 	NetworkPolicy *string `json:"networkPolicy,omitempty"`
 
-	// SSHPublicKey is a string literal containing an ssh public key.
+	// SSHPublicKey is a string literal containing an ssh public key base64 encoded.
```

Comment:
This is a breaking change to the API, but not one that is a type violation. Rather, this is a behavioral breaking change. To bring the API in line with VM and VMSS, we now ask that SSH pub keys be provided base64 encoded. Unfortunately, the APIs for VM / VMSS and AKS differ in that the former requires base64 encoding while the later take a non-encoded pub key.